packages/apis/uniswapv2/src/query/trade.ts

```diff
@@ -392,17 +403,9 @@ function _bestTradeExactOut(
   return bestTrades;
 }
 
-// TODO: export in schema and implement percent difference like uniswap sdk
-export function computePriceImpact(trade: Trade): BigInt {
-  const midPrice = routeMidPrice({
-    route: trade.route,
-  });
-  const midPriceBI = BigInt.fromString(midPrice.amount);
-  const inputAmountBI = BigInt.fromString(trade.inputAmount.amount);
-  const outputAmountBI = BigInt.fromString(trade.outputAmount.amount);
-
-  const exactQuote = midPriceBI.mul(inputAmountBI);
-  return exactQuote.sub(outputAmountBI).div(exactQuote);
+function computePriceImpact(trade: Trade): BigFloat {
```

Comment:
Can you please implement this now as a exported query function?
I think I left this as TODO as we didn't have floats, but should be implemented and exported now I think

Reply:
This is the same as the tradeSlippage function, which is exported